src/Sonarr.Api.V3/Config/MediaManagementConfigModule.cs

```diff
@@ -12,7 +12,7 @@ public MediaManagementConfigModule(IConfigService configService, PathExistsValid
             SharedValidator.RuleFor(c => c.FileChmod).NotEmpty();
             SharedValidator.RuleFor(c => c.FolderChmod).NotEmpty();
             SharedValidator.RuleFor(c => c.RecycleBin).IsValidPath().SetValidator(pathExistsValidator).When(c => !string.IsNullOrWhiteSpace(c.RecycleBin));
-            SharedValidator.RuleFor(c => c.MinimumFreeSpaceWhenImporting).GreaterThan(100);
+            SharedValidator.RuleFor(c => c.MinimumFreeSpaceWhenImporting).GreaterThan(99);
```

Comment:
I think `GreaterThanOrEqualTo` would be better, albeit longer.

Reply:
You're right, that one makes more sense. Updated my PR